x-pack/apm-server/aggregation/spanmetrics/aggregator.go

```diff
@@ -327,11 +365,13 @@ func makeAggregationKey(event *model.APMEvent, resource, targetType, targetName
 		serviceEnvironment: event.Service.Environment,
 		agentName:          event.Agent.Name,
 
+		spanName: spanName,
```

Comment:
Did you consider adding a config option to let users opt-out of span.name aggregation?

Reply:
I did consider it, but would prefer to start out with minimal configuration. I see no reason to opt out except in in the case of high cardinality span names, which is handled automatically by stopping aggregation on `span.name` once the capacity reaches 50%.